src/inference/src/cpp_interfaces/interface/ie_iexecutable_network_internal.cpp

```diff
@@ -91,12 +91,17 @@ void IExecutableNetworkInternal::SetPointerToPlugin(const std::shared_ptr<IInfer
     _plugin = plugin;
 }
 
-void IExecutableNetworkInternal::SetConfig(const std::map<std::string, Parameter>&) {
-    IE_THROW(NotImplemented);
+void IExecutableNetworkInternal::SetConfig(const std::map<std::string, Parameter>& config) {
+    _config = config;
 }
 
-Parameter IExecutableNetworkInternal::GetConfig(const std::string&) const {
-    IE_THROW(NotImplemented);
+Parameter IExecutableNetworkInternal::GetConfig(const std::string& key) const {
```

Comment:
from current cpu/gpu design, this should be implemented by each executable network to return runtime executable network parameters. These configs should be maintained while load network happens without calling setconfig separately.